server/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "scripts": {
+        "start": "concurrently \"cd ../client && npm run dev \" \"node index.js\" "
+    },
+    "dependencies": {
+        "concurrently": "^4.1.0",
```

Comment:
Since concurrently isn't needed for production, it should installed as a dev dependency